src/sonic_ax_impl/mibs/ietf/rfc4363.py

```diff
@@ -85,8 +86,18 @@ def update_data(self):
             if not ent:
                 continue
 
+            bridge_port_id_attr = ""
+            try:
+                bridge_port_id_attr = ent["SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID"]
+            except KeyError as e:
+                # Only write error log once
+                if fdb_str not in self.broken_fdbs:
+                    mibs.logger.error("SyncD 'ASIC_DB' includes invalid FDB_ENTRY '{}': failed to get bridge_port_id, exception: {}".format(fdb_str, e))
```

Comment:
log at warning level, because we skip it and continue with others.

Reply:
Fixed